src/data/CCBV.ml

```diff
@@ -1,80 +1,117 @@
 
-(* This file is free software, part of containers. See file "license" for more details. *)
-
 (** {2 Imperative Bitvectors} *)
 
-let __width = Sys.word_size - 2
+let __width = Sys.word_size - 1
+
+(** We use OCamls ints to store the bits. We index them from the
+    least significant bit. We create masks to zero out the most significant
+    bits that aren't used to store values. *)
+let __lsb_masks =
```

Comment:
Actually, this wasn't performance driven but ease of understanding. I also wanted a clearer way to have all `n` less significant bits set without baroque bit twiddling or a O(n) calculation.

I also played around with having pre-computed masks for the `1 lsl j` in `get|set|flip`'s but that isn't reliably more efficient. Specifically, it improves performance for creating small (relative to universe size) bit sets, but not for bigger cases. I think what happens is that the array used to store these values gets evicted from cache. But I didn't investigate fully.